web/srv/api_handlers.go

```diff
@@ -3,14 +3,18 @@ package srv
 import (
 	"bytes"
 	"encoding/json"
+	"fmt"
 	"io"
 	"net/http"
+	"text/template"
 	"time"
 
 	"github.com/golang/protobuf/jsonpb"
 	"github.com/golang/protobuf/proto"
 	"github.com/gorilla/websocket"
 	"github.com/julienschmidt/httprouter"
+	profileCmd "github.com/linkerd/linkerd2/cli/cmd"
```

Comment:
I'd like to avoid having this package import the `cli/cmd` package. My preference is to move the code that's shared between the CLI and the web server to a separate package in the `pkg` directory.

In fact, as written the web docker image won't build, since we don't add the `cli` directory before building the web binary. All code that the web server depends on needs to live in `web/srv`, `controller`, `pkg`, or `vendor`.